hatch_pip_compile/plugin.py

```diff
@@ -339,3 +340,15 @@ def pipools_environment_dict(self) -> Dict[str, Any]:
         Get the environment dictionary
         """
         return self.metadata.hatch.config.get("envs", {})
+
+    def plugin_check_command(
+        self, command: Union[str, List[str]], *, shell: bool = False, **kwargs: Any
+    ) -> CompletedProcess:
+        """
+        Run a command from the virtualenv
+        """
+        return self.virtual_env.platform.check_command(
+            command=command,
+            shell=shell,
+            **kwargs,
+        )
```

Comment:
Separating this into its own function made this manageable to patch in testing - I'm trying to decide whether to generate a new release for this change since it's the only hatch-pip-compile code change in the PR (🤔 should I create a new release category beyond feature or bug to capture this)